urwid/wimp.py

```diff
@@ -154,36 +155,48 @@ def __init__(
 
           urwid.disconnect_signal(check_box, 'change', callback, user_data)
 
-        >>> CheckBox(u"Confirm")
+        >>> CheckBox("Confirm")
         <CheckBox selectable flow widget 'Confirm' state=False>
-        >>> CheckBox(u"Yogourt", "mixed", True)
+        >>> CheckBox("Yogourt", "mixed", True)
         <CheckBox selectable flow widget 'Yogourt' state='mixed'>
-        >>> cb = CheckBox(u"Extra onions", True)
+        >>> cb = CheckBox("Extra onions", True)
         >>> cb
         <CheckBox selectable flow widget 'Extra onions' state=True>
-        >>> cb.render((20,), focus=True).text # ... = b in Python 3
-        [...'[X] Extra onions    ']
+        >>> cb.render((20,), focus=True).text
+        [b'[X] Extra onions    ']
+        >>> CheckBox("Test", None)
+        Traceback (most recent call last):
+        ...
+        ValueError: None not in (True, False, 'mixed')
         """
-        super().__init__(None) # self.w set by set_state below
-        self._label = Text("")
```

Comment:
we can set all in 1 call instead of chain "construct empty -> call helper to call modifier to replace value from constructor"